src/components/localtunneldebugger/localtunneldebugger.extension.ts

```diff
@@ -0,0 +1,4 @@
+export interface LocalTunnelDebugProvider {
```

Comment:
💡 Idea share for consistency and what others think? since this is the `interface`, we could rename this file to `localTunnelDebuggerProvider` as the interface name is **Or** rename the interface and file as `localtunneldebuggerinterface` 

* Might be worth to at least rename file same as the class name aka: rename from `localtunneldebugger.extension.ts` to `localtunneldebuggerprovider`

Reply:
That's a good point! I will rename the interface & the file to "localTunnelDebuggerProvider" and after I merge your PR. Thanks for pointing that out!

Reply:
@Tatsinnit @daniv-msft  I noticed that the "normal" debugging providers in the same repo (e.g. `dotnetdebugprovider.ts`) use this "debug provider" naming. I think "debug provider" sounds more colloquial than "debugger provider", even though the latter would technically be more consistent. But I'm open to changing it as I want to make sure I'm following the conventions of the repo & keeping things orderly!

Reply:
Sounds good, thanks @amsoedal. Yes, if `Xdebugprovider.ts` is already in use in the codebase, I agree we should follow the existing convention and stick to "debug provider".

Reply:
Cool, thanks for the feedback. I renamed the `localtunneldebugger.extension.ts` file to `localtunneldebugprovider` like you suggested @Tatsinnit, and added the debug option to the deployments like you mentioned offline @daniv-msft. Let me know if there's more changes you guys would like to see or if it looks OK.